src/cloudsc_gpu_atlas/cloudsc_driver_scc_mod.F90

```diff
@@ -182,6 +183,10 @@ SUBROUTINE CLOUDSC_KERNEL( TIMER, NGPTOT, NGPBLKS, NLEV, NPROMA, PTSPHY, &
   call device_allocator%allocate(ZFOEELIQT, [NPROMA, NLEV, NGPBLKS])
 #endif
 
+!#if (ATLAS_HAVE_GPU == 0)
+    CALL IEEE_SET_HALTING_MODE(IEEE_INVALID, .FALSE.)
+!#endif
```

Comment:
```suggestion
    LOGICAL :: LL_HALT_INVALID
    CALL IEEE_GET_HALTING_MODE(IEEE_INVALID, LL_HALT_INVALID)
    CALL IEEE_SET_HALTING_MODE(IEEE_INVALID, .FALSE.)
```
Also please write a little comment stating that this temporarily disables floating-point-trapping of FE_INVALID caused by too aggressive optimisation within the CLOUDSC function.